nodejs/week2/homework/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -0,0 +1,56 @@
+const { response, request } = require("express");
+const express = require("express");
+const router = express.Router();
+
+const meals = require("../data/meals.json");
+
+// 12 tests passed, 6 failed. Can't get my code to work 
+
+router.get("/", async (request, response) => {
+  try {
+    console.log(request.query.maxPrice);
+    response.send(meals);
```

Comment:
Hm, is it just a test route?

Reply:
unfortunately I don remember, I've been stuck here for months. and in my previous class there was a lot of confusion. so maybe I did it to make sure the route is working.